mar/phaser/mar.js

```diff
@@ -803,16 +803,16 @@ function codeListener(message) {
  * Listens for authentications responses from the server
  */
 function authListener(message) {
-
-    if (message.t === "auth") {
-
-        if (message.m === "ok") {
-            console.log("Auth successful");
-            mar.client.requestUserInfo();
-
-        } else {
-            alert("Authentication failed. Please make sure you are logged in and reload the page.");
+    if (message.t === "code") {
```

Comment:
I think this change is supposed to be in codeListener instead of authListener

Reply:
er another mistake. lemme see